client/my-sites/marketing/connections/mailchimp-settings.jsx

```diff
@@ -95,12 +97,12 @@ const MailchimpSettings = ( {
 		<div>
 			<QueryMailchimpLists siteId={ siteId } />
 			<QueryMailchimpSettings siteId={ siteId } />
-			<p>{ translate( 'What Mailchimp list should subscribers be added to?' ) }</p>
+			<p>{ translate( 'What Mailchimp audience should subscribers be added to?' ) }</p>
 			{ Array.isArray( mailchimpLists ) && mailchimpLists.length === 0 && (
 				<Notice
 					status="is-info"
 					text={ translate(
-						"Looks like you've not set up any Mailchimp lists yet. Head to your Mailchimp admin to add a list."
+						"Looks like you've not set up any Mailchimp audiences yet. Head to your Mailchimp admin to add an audience."
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.